src/webextension/firstrun/components/pages.js

```diff
@@ -134,8 +140,11 @@ export class FinishedPage3 extends React.Component {
       name: "manage",
     });
     this.props.next();
+    telemetry.recordEvent("lockbox", "render", "setupDonePage");
+
```

Comment:
As above, this is getting run when the user *clicks* "finish" to close the wizard, but the spec says it "fires after the user authenticates wtih fxa successfully and the message informing them of that *renders*." If you want to follow the spec, create a new method called `componentDidMount()` and run this there.

Also, nit: extra newline